kbase-extension/static/kbase/js/widgets/appWidgets2/validators/resolver.js

```diff
@@ -1,31 +1,45 @@
-define(['require', 'bluebird'], (require, Promise) => {
+define(['require', 'bluebird', '../validation'], (require, Promise, Validator) => {
```

Comment:
This module routes to individual validators based on the parameter spec. Much like the input/view widget resolver, this loads them in dynamically. This is still useful and needed for `struct` and `sequence` inputs (arbitrary parameter groups and lists, respectively). 

This first step toward consolidating everything starts moving toward just calling the proper function in the monolithic validation module. The text one is biggest - future PRs will probably change a number of others at a time, as most of the rest are fairly simple. The `struct` and `sequence` validators might get some extra love later.